### PR TITLE
DOCSUP-3043: Document the null function (ru)

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #14797

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
submodule changed

File: docs/en/sql-reference/table-functions/null.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ toc_title: null function
 
 # null {#null-function}
 
-Accepts an inserted data of the specified structure and immediately drops it away. The function is used for convenience writing tests and demonstrations.
+Creates a temporary table of the specified structure with the [Null table engine](../../engines/table-engines/special/null.md), and according to the Null-engine rules, the table data is ignored and the table itself is immediately droped right after the query execution. The function is used for convenience writing tests and demonstrations.
```
Comment (BayoNet) [inline docs/en/sql-reference/table-functions/null.md:8]:
```suggestion
Creates a temporary table of the specified structure with the [Null](../../engines/table-engines/special/null.md) table engine. According to the `Null`-engine properties, the table data is ignored and the table itself is immediately droped right after the query execution. The function is used for the convenience of test writing and demonstrations.
```

File: docs/en/sql-reference/table-functions/null.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ null('structure')
 
 **Returned value**
 
-A table with the specified structure, which is dropped right after the query execution.
+A temporary Null-engine table with the specified structure.
```
Comment (BayoNet) [inline docs/en/sql-reference/table-functions/null.md:22]:
```suggestion
A temporary `Null`-engine table with the specified structure.
```

File: docs/ru/sql-reference/table-functions/null.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+toc_priority: 53
+toc_title: null функция
+---
+
+# null {#null-function}
+
+Создает временную таблицу указанной структуры [на движке Null](../../engines/table-engines/special/null.md). В соответствии с правилами работы движка Null, данные в таблице игнорируются, а сама таблица удаляется сразу после выполнения запроса. Функция используется для удобства написания тестов и демонстрационных примеров.
```
Comment (BayoNet) [inline docs/ru/sql-reference/table-functions/null.md:8]:
```suggestion
Создает временную таблицу указанной структуры с движком [Null](../../engines/table-engines/special/null.md). В соответствии со свойствами движка, данные в таблице игнорируются, а сама таблица удаляется сразу после выполнения запроса. Функция используется для удобства написания тестов и демонстрационных примеров.
```

File: docs/ru/sql-reference/table-functions/null.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+null('structure')
+```
+
+**Параметр** 
+
+-   `structure` — список колонок и их типов. [String](../../sql-reference/data-types/string.md).
+
+**Возвращаемое значение**
+
+Временная таблица указанной структуры на движке Null.
```
Comment (BayoNet) [inline docs/ru/sql-reference/table-functions/null.md:22]:
```suggestion
Временная таблица указанной структуры с движком `Null`.
```

**Pull request merged**